Composer/packages/client/src/components/AppComponents/SideBar.tsx

```diff
@@ -105,7 +104,7 @@ export const SideBar: React.FC<RouteComponentProps> = () => {
                 iconName={link.iconName}
                 labelName={link.labelName}
                 match={link.match}
-                showTooltip={showTooltips(link)}
+                showTooltip={!link.disabled && !sideBarExpand}
```

Comment:
Are we sure we want to hide the tooltip if the link is disabled?

Reply:
I think we would still want to show the tooltip if a button is disabled. For ex if a button is disabled due to PVA context(qna) we would typically want to know the tooltip instead of expanding the Sidebar

Reply:
That's how the sidebar currently works; the changes in there are a refactor.

Reply:
@mareekuh @vivekkshankar Do either of you have an opinion on rendering the tooltip even if the button is disabled? The current behavior is to hide the tooltip if the button is disabled which seems like an odd behavior. The user might be curious what the button is even if it’s disabled.

Reply:
Yes, please still show it even if the button is disabled.